helper_crates/vtable/src/vrc.rs

```diff
@@ -126,6 +128,12 @@ impl<VTable: VTableMetaDropInPlace + 'static, X> Drop for VRc<VTable, X> {
     }
 }
 
+impl<VTable: VTableMetaDropInPlace + 'static, X> core::fmt::Debug for VRc<VTable, X> {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        f.debug_struct("VRc").field("inner", &unsafe { self.inner.as_ref().data_ptr() }).finish()
```

Comment:
`NonNull` implements `Debug`, so I don't think that you need `unsafe` here

```suggestion
        f.debug_struct("VRc").field("inner", &self.inner).finish()
```